public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/edit.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import { withSelect } from '@wordpress/data';
+import { Component, Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { WC_BLOCKS_STORE } from '../../data';
+import { ScheduleGrid } from './schedule-grid';
+import InspectorControls from './inspector-controls';
+import { ICON } from './index';
+
+const blockData = window.WordCampBlocks.sessions || {};
+
+/**
+ * Top-level component for the editing UI for the block.
+ */
+class ScheduleEdit extends Component {
```

Comment:
Good catch, I think I just did it that way b/c the others are, but we should probably change most of them too?

Reply:
I wouldn't bother changing the other blocks if that's what you mean, but for new components you're adding, sure.

Reply:
Done for this block in b30cca71ff04d2a3d5fa52e4e0649b5d0d4d9bb0

I wasn't planning to do the other blocks in this PR, but we've had a lot of conversations about consistency between all the blocks, so it seems odd to only change this in one of them.

cc @coreymckrill in case you have any thoughts.

Reply:
If I recall, we already converted a bunch of class components to functional in the other blocks when we were iterating on them. But if there's more to do, it's probably worth opening an issue for it. I definitely don't think it should be done as part of the Schedule block.